### DISCUSSION TITLE
Export to PDF

### DISCUSSION THREAD

**Discussion by Calab** [TriliumNext/Trilium#4697] [Ideas]
Would it be possible to export a Trilium notebook as a PDF file? The Trilium tree would be used to generate the "table of contents" for the PDF file. The various Trilium notes could be exported as pages in the PDF file, complete with links within the document.

**Comment by zadam** [comment]
This is not supported unfortunately.

**Comment by Prkl8r** [comment]
I'd like to add a use case for this enhancement.

I keep notes about my home network, truenas, etc. in Trilium. If I could export to a single PDF, I could import into NotebookLM to be able to ask questions and get help with certain issues.
